canonical-websites/snapcraft.io
# Create cache module

# Refactoto S01E03

## Summary

Create module for cache, this is an helper file for the futures api modules we will have in the project.
There is no modification in the logic of the functions, just code moved.

## QA 
- `./run`
- http://0.0.0.0:8004/
- http://0.0.0.0:8004/atom
- http://0.0.0.0:8004/store
- http://0.0.0.0:8004/account
- http://0.0.0.0:8004/account/snaps/SNAP/market
- http://0.0.0.0:8004/account/snaps/SNAP/metrics
- ...
- Try access pages that make API calls and see if it still works
